modules/relation/src/main/JsonView.scala

```diff
@@ -14,4 +15,13 @@ object JsonView {
         "relation" -> r.relation
       ).noNull
     }
+
+  def writeOnlineFriends(friendsOnline: OnlineFriends) = {
+    // We use 'd' for backward compatibility with the mobile client
+    Json.obj(
+      "t" -> "following_onlines",
+      "d" -> friendsOnline.users.map(_.titleName),
+      "playing" -> friendsOnline.usersPlaying
+    ).noNull
```

Comment:
you don't need `noNull` here as none of the values are of the the type `Option[_]`